src/main/java/de/tum/in/www1/artemis/security/SecurityUtils.java

```diff
@@ -141,4 +141,15 @@ public String getName() {
         };
         context.setAuthentication(authentication);
     }
+
+    /**
+     * Get the JWT of the current user.
+     *
+     * @return the JWT of the current user.
+     */
+    public static Optional<String> getCurrentUserJWT() {
```

Comment:
I'd suggest not mentioning the user here.

This method rather returns the JWT for a current security context **if exists**. The context technically doesn't need to reflect a specific user and since the code doesn't mention one, I'd remove the word "user" from this method.